.github/workflows/ci-visibility-performance.yml

```diff
@@ -19,9 +19,7 @@ jobs:
     env:
       ROBOT_CI_GITHUB_PERSONAL_ACCESS_TOKEN: ${{ secrets.ROBOT_CI_GITHUB_PERSONAL_ACCESS_TOKEN }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/node/setup
-      - run: yarn install
-      - uses: ./.github/actions/node/latest
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/node/18
```

Comment:
`node/setup` is using `node@14`: is that the default? They don't say that the docs, as far as I can tell: https://github.com/actions/setup-node#supported-version-syntax. Is 14 faster than 18?

Reply:
Hmm actually this might explain some other issues (for example slowdowns on Windows). Let's leave 18 for now and I'll look into it across the board 👍